com.sleepfiles.OSCAR.yaml

```diff
@@ -0,0 +1,60 @@
+######################################################
+# Flatpak manifest for OSCAR
+# Erico Mendonca <erico.mendonca@gmail.com> Jan 2022 
+######################################################
+
+# Special OBS field because flatpak does not have a version field
+# Default will be '0' if the field is missing.
+#!BuildVersion: 1.3.1
+---
+app-id: com.sleepfiles.OSCAR
+runtime: org.kde.Platform
+runtime-version: '5.15-21.08'
+sdk: org.kde.Sdk
+command: OSCAR
+
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --filesystem=home,xdg-run
```

Comment:
this is an incorrect value. If you need more than one, specify more than one finish args. Also `xdg-run`  doesn't do what you think it does, and require an extra path.

Reply:
It needs to read from SD cards. What would I need to accomplish that?

Reply:
`--filesystem=/media` and `--filesystem=/run/media` should cover it.

Reply:
Done!